### DISCUSSION TITLE
2 issues with qtile

### DISCUSSION THREAD

**Discussion by filippoaceto22** [qtile/qtile#4549] [General]
Hi, i have qtile in wayland mode.
i have 2 problems. When i launch nm-applet --indicator, the widget StatusNotifier show the icon of nm-applet but not work the context menu.

Another issue is that when i m in tiling mode, the context menu of an application go back the nearest window application.
Is possible  to fix this issues?

![image](https://github.com/qtile/qtile/assets/50357161/afb47f9b-57d7-4761-a599-89455ca85417)

**Comment by ervinpopescu** [comment]
1. Use @elParaguayo's [`qtile-extras`](https://github.com/elparaguayo/qtile-extras) repo by substituting `from libqtile import widget` with `from qtile_extras import widget`.
2. I cannot help you with this, as I am not using wayland.
